plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/dev/Stores.js

```diff
@@ -33,56 +35,64 @@ Ext.define('NX.view.dev.Stores', {
     var me = this;
 
     Ext.apply(me, {
```

Comment:
not directly pertinent to this review, but isn't this 'apply' call completely redundant? Just declaring me.items and me.tbar within the initComponent block should be sufficient. I only mention because we have a mix of styles and picking one consistently would aid comprehension.